tests/test_datetime_parsers.py

```
@@ -21,403 +21,375 @@
+                     treq=True, first=True) == (0, 60)
+
+    # Check day_sec_type_in_string, DEPRECATED
+    for include_remainder in (True, False):
+        for info in [
+            ("Time:2000-01-01 00:00:00.00",         0, 0.0, "UTC", ''),
+            ("Time[2000-01-01 00:00:00.00 tai]",    0, 0.0, "TAI", ']'),
+            ("Today is [[2000-01-01 00:00:00 ET]]", 0, 0.0, "TDB", ']]'),
+            ("2000-01-01 00:00 TDT was today",      0, 0.0, "TT",  ' was today'),
+        ]:
+
+            (test, day, sec, tsys, expected_remainder) = info
```

Ruff (0.15.6)

[warning] Unpacked variable `expected_remainder` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)